packages/ui/src/components/Dialog/Dialog.tsx

```diff
@@ -120,7 +120,8 @@ const DialogHeader = ({
 }: DialogProps & SizeProp & PropsWithChildren) => (
   <div
     className={twMerge(
-      "flex items-center justify-between p-3 md:p-4",
+      "flex items-center p-3 md:p-4",
+      children ? "justify-between" : "justify-end",
```

Comment:
Why not "justify-end" as default? We area supposing that if the dev passes a child component, he wants to use "justify-between". Should we do that? Is this 100% of our cases?

Reply:
we should, cause it's a good default. And it 100% of our cases.

if we left the `justify-end` by default this would happen.

<img width="447" alt="image" src="https://github.com/user-attachments/assets/e009e049-0b7d-4641-a69b-b94763d8e1c2">


The case here would be to ignore the justify end, and live with the arrow on the left. But the `DialogClose` is indeed better at the right.

Reply:
In this case, the dev should use the "justify-between" class, right? Like he normally would with a default flex component. Not a big fan of "magic" happening on the background, but if this works good for us, let's merge it.

Reply:
I see your concerns, but that would require the dev to always have to add that class. I'll go with it now.